src/main/java/com/mineboundteam/minebound/magic/OffensiveSpells/NecroticOffensiveSpell.java

```diff
@@ -79,7 +79,7 @@ public static void onAttack(LivingDamageEvent event){
                 spell = player.getUseItem();
             else {
                 spell = getSelectedSpell(player, PlayerSelectedSpellsProvider.PRIMARY_SPELL);
-                if(spell.isEmpty())
+                if(spell.isEmpty() || !(spell.getItem() instanceof NecroticOffensiveSpell))
```

Comment:
The only issue with this is it won't behave as intended if multiple Necrotic Offensives are equipped and the secondary one is active. Might as well also check the active tag.

Reply:
Okay _now_ it's fixed. We should let players use both hands if they want